README.md

```diff
@@ -214,6 +216,32 @@ customMetric(String name, long value)
 Calling either of these will add a custom metric with the specified name and
 the given value. Custom metric names are limited to 128 characters.
 
+## Event Info
+
+This plugin records input event types and includes in the report the origin
+of certain events.
+
+It operates on the given input classes:
+
+ * `com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent`
+ * `com.amazonaws.services.lambda.runtime.events.CloudFrontEvent`
+ * `com.amazonaws.services.lambda.runtime.events.KinesisEvent`
+ * `com.amazonaws.services.lambda.runtime.events.KinesisFirehoseEvent`
+ * `com.amazonaws.services.lambda.runtime.events.S3Event`
+ * `com.amazonaws.services.lambda.runtime.events.ScheduledEvent`
+ * `com.amazonaws.services.lambda.runtime.events.SNSEvent`
+ * `com.amazonaws.services.s3.event.S3EventNotification`
+
+By default this plugin is enabled and requires no changes to your code unless
```

Comment:
Is there an option to disable the plugin via environment variable? Or how would a user "turn it off" -- define their plugins so that the event-info isn't included in their plugins. Let's give those instructions here.

Reply:
Don't believe the Node and Python agents use this environment variable currently.

Reply:
Are they enabled programmatically?

Reply:
Could Java be EventInfo versus event-info and thus IOPIPE_EVENTINFO_ENABLE?

Reply:
@kolanos @XerTheSquirrel the other agent's don't use the environment variable because the event-info of those have no way to be turned off -- the event info is either included or excluded as a plugin. Original question is how a user would exclude event-info from their plugins if they so chose, not necessarily asking for the environment variable

Reply:
And to the programmatically question: event-info is enabled by including it in the config object, so to that degree, I believe the answer is yes, programattically.

Reply:
@pselle For the name, the plugin could be called anything although anything besides `event-info` would be inconsistent with the other agents. I am guessing that if the `profiler` and `trace` plugins were renamed that the dashboard would have no clue what they are actually for, I do not know the internals of the dashboard but with different names in an agent it is possible to make an oversight which misses the Java plugin.

In the Java agent right now you cannot exclude the plugin from existing unless you were to remove it from the repository and force the user to include it as a dependency (and they would need to write their maven config _correctly_ otherwise it will just not work and probably make every other external plugin not working either). However, it would not be like a single line addition it would be like using a bunch of XML to get it working.

An alternative would there being some kind of `iopipe.config` in the JAR which can be used to setup compile time static configuration settings, but there are environment variables that would be doing the same thing not to mention that redeploys would be needed to change these now statically included settings.

Reply:
Do we have an answer on the configuration question? If someone _didn't_ want to include event-info in their data, can they turn it off somehow? If this is quite complex, we can evaluate if we want to always include event-info in Java and visit turning it off when it comes up.

As to the name, what the plugin calls itself is a string, so I'm unsure why that's an issue, I thought we were talking about class names.

Reply:
@pselle Yes with #62 , setting`IOPIPE_EVENT_INFO_ENABLE` to `false` can be used to disable the plugin. In the report it just does this: `{"name":"event-info","version":"1.2.0","homepage":"https://github.com/iopipe/iopipe-java","enabled":false}`.

Reply:
And does not report anything about the event that just happens as none of the code runs at all.